<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelTableWidget</class>
 <widget class="QWidget" name="ModelTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
  <action name="add1Action">
   <property name="text">
    <string>窗口1</string>
   </property>
  </action>
  <action name="add2action">
   <property name="text">
    <string>窗口2</string>
   </property>
  </action>
  <action name="add3action">
   <property name="text">
    <string>窗口3</string>
   </property>
  </action>
  <action name="add4action">
   <property name="text">
    <string>窗口4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
